### DISCUSSION TITLE
wiki.js v3 actual release date is?

### DISCUSSION THREAD

**Discussion by jdarrelljr** [requarks/wiki#5160] [General]
With today being April 6th, I have steadily checked in with the wiki.js website to see if v3 has an actual release date. Unfortunately I do not find one. So my question again is, What is the release date for v3.0?

As we are in the middle of planning for the second half of 2022, it would be great if someone could respond.

Please note: No one responded to my previous inquiry from late 2021. I know that I am not the only one wanting to know.

Thank You!
Jack

**Comment by NGPixel** [comment]
https://docs.requarks.io/releases
